src/cb_events/__init__.py

```
@@ -26,6 +27,8 @@
 except PackageNotFoundError:  # pragma: no cover
     __version__ = "0.0.0"
 
+logging.getLogger(__name__).addHandler(logging.NullHandler())  # ruff:ignore[non-empty-init-module] # Configuration of handlers is the prerogative of end-users
```

Ruff (0.16.0)

[warning] Invalid rule code in suppression: non-empty-init-module

Enable `lint.preview` to use rule names

(RUF102)